.gitlab-ci.yml

```diff
@@ -30,8 +30,5 @@ test:acceptance_tests:
     DOCKER_HOST: tcp://docker:2375/
     DOCKER_DRIVER: overlay2
 
-test:static:
-  image: golang:1.14
-
 test:unit:
   image: golang:1.14
```

Comment:
Can we also remove this one? I did so for `mender-cli`, see https://github.com/mendersoftware/mender-cli/pull/137

Reply:
no, as described in the commit message:
> - only for static tests though as unit tests still need to be aligned with go:1.14

this is due to the certificate in mendersoftware/integration#1253 - to be correct, even the static check change should not be adjusted - but I think it is tolerable... for the unit tests I don't think it is

Reply:
Sorry I didn't read that :see_no_evil: